javascript/big-practice/src/scripts/views/noteView.js

```diff
@@ -43,32 +44,55 @@ export default class NoteView {
   renderNoteForm() {
     const formElement = document.createElement('div');
     formElement.setAttribute('class', 'overlay');
+    const note = this.noteItem;
 
     formElement.innerHTML = `
-      <form class="form-note note-form-overlay" id="${this.noteItem.id}">
+      <form class="form-note note-form-overlay" id="${note.id}">
         <div class="form-title">
           <div class="form-group">
-            <textarea class="input-note note-title" rows="1" placeholder="Title">${this.noteItem.title}</textarea>
+            <textarea class="input-note note-title" rows="1" placeholder="Title">${note.title}</textarea>
             <figure class="icon-pin-cover">
               <img src="${iconPin}" alt="icon pin">
             </figure>
           </div>
         </div>
         <div class="form-group">
-          <textarea class="input-note note-description" rows="1" placeholder="Take a note...">${this.noteItem.description}</textarea>
+          <textarea class="input-note note-description" rows="1" placeholder="Take a note...">${note.description}</textarea>
         </div>
         <div class="form-utilities">
           <div class="form-group">
             <div class="items-utilities">
               <img src="${iconColorBoard}" alt="icon color board">
             </div>
             <div class="form-buttons">
-              <button class="btn btn-delete-form" type="button" data-id="${this.noteItem.id}">Delete</button>
-              <button class="btn btn-close" type="button" data-id="${this.noteItem.id}">Close</button>
+              <button class="btn btn-delete-form" type="button" data-id="${note.id}">Delete</button>
+              <button class="btn btn-close" type="button" data-id="${note.id}">Close</button>
             </div>
           </div>
         </div>
       </form>`;
     return formElement;
   }
+
+  /**
+   * @description function create confirm message
+   *
+   * @returns {Object} confirmMessage
+   */
+  renderConfirmPopup() {
```

Comment:
We should define a common function for this case. In the case we have another confirm popup then we can reuse it.